public/contentStyle.css

```
@@ -1,66 +1,20 @@
+	border-radius: 8px !important;
+	background-color: var(--yt-spec-overlay-background-medium-light, rgba(0, 0, 0, 0.3)) !important;
+	text-shadow: 0 0 2px #000 !important;
 }
 
-body.no-scroll .yte-button-tooltip {
-	font-size: 20px !important;
+.ytp-big-mode .yte-button-tooltip {
 	line-height: 22px !important;
-	padding: 8px 9px !important;
-	transform: translate(-50%, -75%) !important;
+	font: 20px !important;
```

Biome (2.1.2)

[error] Unexpected shorthand property font after line-height



(lint/suspicious/noShorthandPropertyOverrides)